### ISSUE TITLE
[POS] Horizontal scrollable filters on mobile

### ISSUE THREAD

**Issue by dstrukt** [btcpayserver/btcpayserver#5255]
Takes up a considerable amount of space when there are more than 4-8 filters, and could hypothetically cover the whole screen with 12(?)+. Additionally, with search + keyboard, an item is barely visible as well.

We should consider a horizontal scrolling filter implementation OR potentially a dropdown/drawer selector if the horizontal scroller proves to be too finicky on a mobile browser.

![IMG_0613](https://github.com/btcpayserver/btcpayserver/assets/6250771/54bf05c1-24b5-47a6-8c89-5fbc23934282)

**Comment by dennisreimann** [comment]
Yup, I tried it — the only negative aspect is that we cannot have it scrollable with the items being centered. I tried various things, it simply doesn't work and the items need to be left aligned, which looks weird if there aren't enough of them. 

Making it dependent on the number of items won't also work, because of the different (text) lengths of the items and the number would vary across breakpoints.

I'd rather vote to make it a dropdown if there are more than five items.
